### PR TITLE
feat[vitess]: Update connector properties for debezium-1.9.7

### PR DESCRIPTION
**Pull request by s-gelazevicius**
After debezium upgrade some optional vitess config parameters where added and some removed. 
Making config parameters up to date with the latest supported debezium version: 1.9.7

### CODE REVIEW

File: flink-connector-vitess-cdc/src/main/java/com/ververica/cdc/connectors/vitess/table/VitessTableFactory.java [hunk 1]
```diff
@@ -136,13 +145,12 @@ public DynamicTableSource createDynamicTableSource(DynamicTableFactory.Context c
...
         String username = config.get(USERNAME);
         String password = config.get(PASSWORD);
-        VtctldConfig vtctldConfig =
-                new VtctldConfig.Builder()
-                        .hostname(config.get(VTCTL_HOSTNAME))
-                        .port(config.get(VTCTL_PORT))
-                        .username(config.get(VTCTL_USERNAME))
-                        .password(config.get(VTCTL_PASSWORD))
-                        .build();
+        String shard = config.get(SHARD);
```
Comment (ruanhang1993) [inline flink-connector-vitess-cdc/src/main/java/com/ververica/cdc/connectors/vitess/table/VitessTableFactory.java:148]: For an option which does not have a default value, we should get its value as following.
```java
String shard =config.getOptional(SHARD).orElse(null);
```
Besides this, we should add `@Nullable` in the constructor and the field to remind the developer.
```java
@Nullable
private final String shard;
public VitessTableSource(
            ResolvedSchema physicalSchema,
            int port,
            String hostname,
            String keyspace,
            String tableName,
            String username,
            String password,
            @Nullable String shard,
            String gtid,
            Boolean stopOnReshard,
            Boolean tombstonesOnDelete,
            SchemaAdjustmentMode schemaNameAdjustmentMode,
            TabletType tabletType,
            String pluginName,
            String name,
            Properties dbzProperties) {
......
}
```

File: flink-connector-vitess-cdc/src/main/java/com/ververica/cdc/connectors/vitess/table/VitessTableSource.java [hunk 2]
```diff
@@ -54,7 +54,11 @@ public class VitessTableSource implements ScanTableSource {
     private final String username;
     private final String password;
     private final String tableName;
-    private final VtctldConfig vtctldConfig;
+    private String shard;
```
Comment (ruanhang1993) [inline flink-connector-vitess-cdc/src/main/java/com/ververica/cdc/connectors/vitess/table/VitessTableSource.java:57]: `@Nullable`

File: flink-connector-vitess-cdc/src/main/java/com/ververica/cdc/connectors/vitess/table/VitessTableSource.java [hunk 3]
```diff
@@ -54,7 +54,11 @@ public class VitessTableSource implements ScanTableSource {
     private final String username;
     private final String password;
```
Comment (ruanhang1993) [inline flink-connector-vitess-cdc/src/main/java/com/ververica/cdc/connectors/vitess/table/VitessTableSource.java:55]: `@Nullable`

File: flink-connector-vitess-cdc/src/main/java/com/ververica/cdc/connectors/vitess/table/VitessTableSource.java [hunk 4]
```diff
@@ -66,7 +70,11 @@ public VitessTableSource(
             String tableName,
             String username,
```
Comment (ruanhang1993) [inline flink-connector-vitess-cdc/src/main/java/com/ververica/cdc/connectors/vitess/table/VitessTableSource.java:71]: `@Nullable`

File: flink-connector-vitess-cdc/src/main/java/com/ververica/cdc/connectors/vitess/table/VitessTableSource.java [hunk 5]
```diff
@@ -212,8 +240,18 @@ public String toString() {
                 + ", tableName='"
                 + tableName
                 + '\''
-                + ", vtctldConfig="
-                + vtctldConfig
+                + ", shard='"
+                + shard
```
Comment (ruanhang1993) [inline flink-connector-vitess-cdc/src/main/java/com/ververica/cdc/connectors/vitess/table/VitessTableSource.java:244]: These nullable fields will cause the NullPointerException. Please use like this.
```java
String.valueOf(shard)
```

**Review by ruanhang1993 — COMMENTED** [review]
@s-gelazevicius Thanks for changes. I left some comments.

Please update the docs for these new options.

**Review by ruanhang1993 — APPROVED** [review]

**Comment by gtk96** [comment]
image

docs there are some display anomalies, as shown above, can this be fixed together?

The redundant characters cause problems with the display after the page is built.

image

**Comment by gtk96** [comment]
> > image
> > docs there are some display anomalies, as shown above, can this be fixed together?
> > The redundant characters cause problems with the display after the page is built.
> > image
> 
> @gtk96 https://github.com/ververica/flink-cdc-connectors/blob/fc5188f5f35a329695ea2bee0df59e88545532e1/docs/content/connectors/vitess-cdc.md It looks well , do you use latest code?

There will be problems when converting to an html page.

https://ververica.github.io/flink-cdc-connectors/master/content/connectors/vitess-cdc.html#connector-options

image

**Comment by GOODBOY008** [comment]
@s-gelazevicius I foud docs/content/connectors/vitess-cdc.md remove `&ldquo;` and `&rdquo` in file.
}@}353F4~KBZ0N8V5AU)KQM

**Review by GOODBOY008 — COMMENTED** [review]
@s-gelazevicius Please fix `docs/content/connectors/vitess-cdc.md`.

File: docs/content/connectors/vitess-cdc.md [hunk 6]
```diff
@@ -69,15 +67,15 @@ Connector Options
 ----------------
 
 
-<div class="&ldquo;highlight&rdquo;">
-    <table class="&ldquo;colwidths-auto">
+<div class="highlight;">
```
Comment (gtk96) [inline docs/content/connectors/vitess-cdc.md:70]:
```suggestion
<div class="highlight">
```
image

Is the code here necessary?

**Review by gtk96 — APPROVED** [review]
Thanks @s-gelazevicius LGTM

**Review by GOODBOY008 — APPROVED** [review]
LGTM

**Pull request merged**